pkgs/development/libraries/tiledb/default.nix

```diff
@@ -0,0 +1,32 @@
+{ pkgs ? import ./pkgs.nix }:
```

Comment:
The dependency structure should follow other similar packages.

For example gumbo has this:

```
  { stdenv, fetchFromGitHub, autoconf, automake, libtool }:
  
  stdenv.mkDerivation rec {
  ...
  }
```

Reply:
ok.